ServiceApp/src/org/droidplanner/services/android/core/gcs/follow/Follow.java

```diff
@@ -121,13 +170,29 @@ public void onDroneEvent(DroneEventsType event, MavLinkDrone drone) {
         }
     }
 
+    public void onFollowNewLocation(android.location.Location location) {
+        Timber.d("onFollowNewLocation(%s)", location);
```

Comment:
@kellyschrock here we should check that the location source is `EXTERNAL` before passing along the received location.